File: src/test/java/org/jdc/template/ux/individual/IndividualPresenterTestModule.java
package org.jdc.template.ux.individual;

import org.dbtools.android.domain.config.DatabaseConfig;
import org.jdc.template.model.database.TestMainDatabaseConfig;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;

@Module
public class IndividualPresenterTestModule {
    @Provides
    @Singleton
    DatabaseConfig provideDatabaseConfig() {
        return TestMainDatabaseConfig.getInstance();
    }
}

File: src/test/kotlin/org/jdc/template/ux/individual/IndividualPresenterTestModule.kt
package org.jdc.template.ux.individual

import dagger.Module
import dagger.Provides
import org.dbtools.android.domain.config.DatabaseConfig
import org.jdc.template.model.database.TestMainDatabaseConfig
import javax.inject.Singleton

// Test Module
@Module
class IndividualPresenterTestModule {
    @Provides
    @Singleton
    fun provideDatabaseConfig(): DatabaseConfig {
        return TestMainDatabaseConfig.instance
    }
}